app/src/main/contrib/components/Responsibilities/BurdenEstimates/UploadBurdenEstimatesForm.tsx

```diff
@@ -14,48 +17,32 @@ export interface UploadBurdenEstimatesFormComponentProps {
     canCreate: boolean;
 }
 
-interface State {
-    hasUploadSuccess: boolean;
-    errors: ErrorInfo[]
-
+export interface UploadBurdenEstimatesFormComponentProps extends UploadBurdenEstimatesFormPublicProps {
+    hasSuccess: boolean;
+    errors: ErrorInfo[];
+    resetPopulateState: () => void;
 }
 
-export class UploadBurdenEstimatesForm extends React.Component<UploadBurdenEstimatesFormComponentProps, State> {
-    private uploadSuccessMessage = "Success! You have uploaded a new set of burden estimates";
-
-    constructor(props: UploadBurdenEstimatesFormComponentProps) {
-        super(props);
-        const result = helpers.ingestQueryStringAndReturnResult();
```

Comment:
This is a vestige from the redirect flow where the user was redirected to this page with error/success messages encoded in a query string. Success/error results are now populated my mapping the global state to props below.